lib/AddonSignals.py

```diff
@@ -1,135 +1,162 @@
 # -*- coding: utf-8 -*-
+# GNU Lesser General Public License v2.1 (see COPYING or https://www.gnu.org/licenses/gpl-2.1.txt)
```

Comment:
this not follow SPDX convention

Reply:
Comments in code do not use SPDX. SPDX identifiers are used for computer-processed fields not for human-processed information (license one-liners are intended for humans).

Reply:
I don't understand what you mean, but this does not follow the SPDX directives at all, if you do a test with various check software will not be approved.

You can see how to do in netflix repo,
even licenses should not be written with links, but placed in the folder LICENSES,
otherwise it is useless to follow the SPDX directives...

Reply:
Make a new PR for your change if you like. I am not interested to take it this far.

The license is in LICENSE.txt, I don't see the point of an additional LICENSES/ directory if we have a single license anyway. (The license is 5x larger than the library, so having the license in the source tree twice is unneeded and unwanted IMO).

Again, conventions are nice, up to a point and then they become pedantic :smile:

Reply:
it is addon checker, devs want to go in this way...
agree that this project is big like a flea

Reply:
AFAIK this was only related to addon.xml, not code comments in your source files.